walkthroughs/week-5-tdd/project/src/main/java/com/google/sps/Utils.java

```diff
@@ -0,0 +1,15 @@
+package com.google.sps;
+
+import java.util.Set;
+
+public class Utils {
```

Comment:
actually avoid kitchen-sink Utils class. This method can be moved back into the code that uses it, specially it's just doing a set intersection. I'm actually pretty sure you can do that without creating a method something like eventAttendees.intersect(requestAttendes) or something like that.

Reply:
Fixed